nixos/modules/system/boot/luksroot.nix

```diff
@@ -433,7 +433,11 @@ let
           echo "Please move your mouse to create needed randomness."
         ''}
           echo "Waiting for your FIDO2 device..."
-          fido2luks open${optionalString dev.allowDiscards " --allow-discards"} ${dev.device} ${dev.name} ${dev.fido2.credential} --await-dev ${toString dev.fido2.gracePeriod} --salt string:$passphrase
+          ${if dev.fido2.credential != null then ''
```

Comment:
The option credential has higher priority than option credentials, which in not expected and documented.